<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radkfile interesect query</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>clear input</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <family>IPAPMincho</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="toolTip">
      <string>paste kanjis here, and all the key belonging to at least one of them will highlight</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>clear table</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <family>Bitstream Cyberbit</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>13</number>
     </property>
     <property name="columnCount">
      <number>20</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit">
     <property name="font">
      <font>
       <family>Bitstream Cyberbit</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="toolTip">
      <string>display all the kanjis who contains all of those keys selected on the above table</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>tableWidget</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
